node/src/db/db_handlers.rs

```diff
@@ -0,0 +1,1140 @@
+                    .get(&*parent_bead_hash)
+                    .unwrap();
+                if let Some(value) = braid_parent_set.get_mut(&bead.0) {
+                    value.insert(*current_parent_bead_index);
+                }
+            }
+        }
+        //Constructing ancestor set, children set will be empty as it will become the next tip
+        let mut ancestor_mapping: HashMap<usize, HashSet<usize>> = HashMap::new();
+        consensus_functions::updating_ancestors(
+            &braid_data,
+            bead.block_header.block_hash(),
+            &mut ancestor_mapping,
+            &braid_parent_set,
+        );
+        let bead_id = braid_data
+            .bead_index_mapping
+            .get(&bead.block_header.block_hash())
+            .unwrap();
+        //Considering the index of the beads in braid will be same as the (insertion ids-1)
+        let current_bead_parent_set = braid_parent_set.get(&(bead_id)).unwrap();
+        let mut relative_tuples: Vec<RelativeTuple> = Vec::new();
+        let mut parent_timestamp_tuples: Vec<ParentTimestampTuple> = Vec::new();
+        //Constructing relatives and parent_timestamps
+        for parent_bead in current_bead_parent_set {
+            relative_tuples.push(RelativeTuple {
+                parent: (*parent_bead as i64) + 1,
+                child: (*bead_id as i64) + 1,
+            });
+            let current_parent_timestamp = braid_data
+                .beads
+                .get(*parent_bead)
+                .unwrap()
+                .committed_metadata
+                .start_timestamp;
+            parent_timestamp_tuples.push(ParentTimestampTuple {
+                parent: (*parent_bead as i64) + 1,
+                child: (*bead_id as i64) + 1,
+                timestamp: current_parent_timestamp.to_u32().to_i64().unwrap(),
+            });
+        }
+        let mut transaction_tuples: Vec<TransactionTuple> = Vec::new();
+        for bead_tx in bead.committed_metadata.transactions.iter() {
+            transaction_tuples.push(TransactionTuple {
+                bead_id: (*bead_id as i64) + 1,
+                txid: bead_tx.compute_txid().to_string(),
+            });
+        }
+        let transactions_values = transaction_tuples
+            .iter()
+            .map(|t| format!("(last_insert_rowid(), '{}')", t.txid))
+            .collect::<Vec<_>>()
+            .join(", ");
+
+        let parent_timestamps_values = parent_timestamp_tuples
+            .iter()
+            .map(|p| format!("({}, {}, {})", p.parent, p.child, p.timestamp))
+            .collect::<Vec<_>>()
+            .join(", ");
+
+        let relatives_values = relative_tuples
+            .iter()
+            .map(|r| format!("({}, {})", r.child, r.parent))
+            .collect::<Vec<_>>()
+            .join(", ");
+
+        log::info!("{:?} ---- {:?}", relatives_values, parent_timestamps_values);
+
+        //Begin the insertion transaction
+        let sub_query_1 = "
```

Comment:
Please merge the sub_queries into a single string query instead of assembling the query as a string on line 167. Easier to see what's going on but logically equivalent.